moban/utils.py

```diff
@@ -5,10 +5,10 @@
 import stat
 import errno
 
-from ruamel.yaml import YAML
 import moban.reporter as reporter
 import moban.constants as constants
 import moban.exceptions as exceptions
+from ruamel.yaml import YAML
```

Comment:
Just a little improvement here. Since `ruamel.yaml` is a third party dependency, it should be placed in between `import errno` and `import moban.reporter import YAML`. ([ref](https://www.python.org/dev/peps/pep-0008/#imports))

Reply:
hmm... `make format` put the import at that position. which should i follow?

Reply:
.isort.cfg needs updating.

here is the isort docs: https://isort.readthedocs.io